src/pages/what-is-ebpf.js

```diff
@@ -616,6 +616,11 @@ class Description extends React.Component {
           <h4>Cilium</h4>
           <ul>
             <Reference
+              link="https://linuxplumbersconf.org/event/7/contributions/674/"
+              name="Kubernetes Service Load-Balancing at Scale with BPF & XDP"
+              description="Daniel Borkmann & Martynas Pumputis, Linux Plumbers 2020"
```

Comment:
Other entries have the format:
```suggestion
              description="Daniel Borkmann & Martynas Pumputis, Linux Plumbers, Aug 2020"
```

Reply:
Makes sense, added, though not all of them have it.

Reply:
Ah, only checked the few right above :-)